RevenueCatUI/PaywallView.swift

```diff
@@ -83,8 +89,9 @@ public struct PaywallView: View {
     private var content: some View {
         VStack { // Necessary to work around FB12674350 and FB12787354
             if let checker = self.introEligibility, let purchaseHandler = self.purchaseHandler {
-                if let offering = self.offering {
+                if let offering = self.offering, let customerInfo = self.customerInfo {
                     self.paywallView(for: offering,
+                                     activelySubscribedProductIdentifiers: customerInfo.activeSubscriptions,
```

Comment:
This _may_ not actually be enough information to accurately show a purchased products with paywalls due to cross platform 😬 

One case for this is if your offering looks like...
- Monthly
  -  Apple - `pro_annually`
  - Google - `pro_yearly`
  
If the user is logged in and sharing purchases a crossed platforms, the paywall would incorrectly show their state and they _could_ accidentally purchase another subscription. 

It _could_ be on the developer to make sure the paywall is shown correctly but it still could happen 🤷‍♂️ 

The thing with this is... I don't think we have a solution for this right now 😅 We would essentially need a way to get "packages purchased" from the customer info so that it correctly maps to the packages that we are showing in the Paywall UI

Reply:
Yeah that's right... 
Given that this is just a "nice to have" I don't think it's a big deal if we don't detect that particular scenario.